library/ix-dev/community/netbootxyz/templates/_persistence.tpl

```diff
@@ -0,0 +1,27 @@
+{{- define "netboot.persistence" -}}
+persistence:
+  config:
+    enabled: true
+    {{- include "ix.v1.common.app.storageOptions" (dict "storage" .Values.netbootStorage.config) | nindent 4 }}
+    targetSelector:
+      netboot:
+        netboot:
+          mountPath: /config
+        {{- if and (eq .Values.netbootStorage.config.type "ixVolume")
+                  (not (.Values.netbootStorage.config.ixVolumeConfig | default dict).aclEnable) }}
+        01-permissions:
+          mountPath: /mnt/directories/netbootxyz/config
```

Comment:
```suggestion
          mountPath: /mnt/directories/config
```
I don't remember if the script handles correctly nested dirs, lets keep it single level